pkg/kubectl/service_basic_test.go

```diff
@@ -58,13 +58,6 @@ func TestServiceBasicGenerate(t *testing.T) {
 			expectErr:   true,
 		},
 		{
-			name:        "clusterip-none and port mapping",
```

Comment:
"identity mapped" means X -> X, Y -> Y.  BUt as above, this would be a break, so just comment why we don't enforce it.